python/ray/dashboard/client/src/pages/dashboard/node-info/features/Disk.tsx

```diff
@@ -23,15 +26,27 @@ export const ClusterDisk: ClusterFeatureComponent = ({ nodes }) => {
   );
 };
 
-export const NodeDisk: NodeFeatureComponent = ({ node }) => (
+export const NodeDisk: NodeFeatureRenderFn = ({ node }) => (
   <UsageBar
     percent={(100 * node.disk["/"].used) / node.disk["/"].total}
     text={formatUsage(node.disk["/"].used, node.disk["/"].total, "gibibyte")}
   />
 );
 
-export const WorkerDisk: WorkerFeatureComponent = () => (
+export const nodeDiskAccessor: Accessor<NodeFeatureData> = ({ node }) =>
+  node.disk["/"].used;
```

Comment:
Can you add a TODO here to make it configurable? I remember someone asked a feature request for this.

Reply:
I'm not sure this is the best place to track it. Do you know if it has an issue already? If not, I can create one.